stonesoup/models/clutter/tests/test_models.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+import pytest
+from pytest import approx
+from scipy.stats import multivariate_normal
+import datetime
+
+from ..clutter import ClutterModel
+from ...measurement.nonlinear import CartesianToElevationBearingRange
+from ....types.detection import Clutter
+from ....types.array import StateVector
+from ....types.state import State
+
+@pytest.mark.parametrize(
+    "model_class",
+    [ClutterModel]
+)
+@pytest.mark.parametrize(
+    "ModelClass, clutter_rate, dist,\
+     dist_params",
+    [
+        (   # 2D meas
+            ClutterModel,
+            5,
+            np.random.default_rng().uniform,
+            ((-50, 50), (-50, 50)),
+        ),
+        (   # 3D meas
+            ClutterModel,
+            5,
+            np.random.default_rng().uniform,
+            ((-50, 50), (-50, 50), (-50, 50)),
+        )
+    ],
+    ids=["Clutter2D", "Clutter3D"]
+)
+def test_model(ModelClass, clutter_rate, dist, dist_params):
+    model_test = ModelClass(clutter_rate=clutter_rate,
+                            distribution=dist,
+                            dist_params=dist_params)
+
+    meas_model = CartesianToElevationBearingRange(
+            ndim_state=6,
+            mapping=(0, 2, 4),
+            noise_covar=np.eye(3))
+    model_test.measurement_model = meas_model
+
+    truth = State(StateVector([1, 1, 1]), timestamp=datetime.now)
+    clutter = model_test.function(truth)

```

Comment:
That's a good point. I can see it either way. The measurement models take a single state and return a single state. The clutter model will still return a set of states. The prototype started as a ClutterSensor and so I used the same parameters as the [`Sensor` parent class. ](https://github.com/ekhunter123/Stone-Soup/blob/main/stonesoup/models/clutter/clutter.py). I never got around to changing it since then. As the ground truth is only used for a timestamp it could even be simplified to just take in a timestamp, rather than passing a single (or set of) state(s).

What do you think would fit best with the other models? It would be nice if they all had roughly the same parameters/design.